discovery/gossiper.go

```diff
@@ -512,6 +512,22 @@ func (d *AuthenticatedGossiper) ProcessRemoteAnnouncement(msg lnwire.Message,
 
 		errChan <- nil
 		return errChan
+
+	// To avoid inserting edges in the graph for our own channels that we
+	// have already closed, we ignore such channel announcements coming
+	// from the remote.
+	case *lnwire.ChannelAnnouncement:
```

Comment:
Interesting, I guess this could happen after SCB recovery, the channel isn't yet closed, but we learn about it? I wonder why this didn't show up in any of the SCB itests.

Reply:
I think it can also happen just after you have closed a channel, then someone send you the announcement?